openapi/src/main/resources/templates/redoc/index.html

```diff
@@ -16,7 +16,7 @@
     <body>
         {{rapipdf.tag}}
         <redoc id='redoc' {{redoc.attributes}}></redoc>
-        <script src='{{redoc.js.url}}'></script>
+        <script src='{{redoc.js.url.prefix}}redoc.standalone.js'></script>
```

Comment:
I don't love this - seems like we're forcing a certain file name structure here when it might change at any point.

Reply:
Maybe, but this is standard redoc/swagger-ui/rapidoc tmplates structure.his may change, but only in future releases of these interfaces. You can always make custom templates, but for standard templates, in my opinion, everything is done correctly